### DISCUSSION TITLE
2009/05/01/%E7%BB%88%E4%BA%8E%E4%BA%A4%E7%A8%BF%E4%BA%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#255] [Announcements]
# 2009/05/01/%E7%BB%88%E4%BA%8E%E4%BA%A4%E7%A8%BF%E4%BA%86

五一放假前终于把书稿寄给了出版社，心情一下子轻松了很多。

https://blog.qizhen.xyz/2009/05/01/%E7%BB%88%E4%BA%8E%E4%BA%A4%E7%A8%BF%E4%BA%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Coral
2009年05月5日 09:59 编辑
终于还真出书了。恭喜！

回复
xiaoji
2009年05月4日 14:48 编辑
啥时一起吃个饭吧，祝贺一下:)

回复
zomby
2009年05月4日 13:28 编辑
恭喜！期待已久

回复
心花怒放
2009年05月4日 08:19 编辑
要是有电子版的就好了

回复
Jun
2009年05月4日 01:04 编辑
恭喜恭喜！快点出版吧～

回复
帆
2009年05月2日 12:10 编辑
山东人民发来贺电！

回复
Yigang
2009年05月1日 23:26 编辑
6月底回国，能赶上你的签名售书么？呵呵

回复
Wu
2009年05月1日 22:37 编辑
恭喜~

回复
味道
2009年05月1日 20:54 编辑
恭喜恭喜！！！来自enjoybitter的祝贺！！

回复
Haizi
2009年05月1日 20:30 编辑
Congratulations!一直关注的你space，获益匪浅。希望能够早一点看到你的新书面世，谢谢！

回复
奕瑋
2009年05月1日 17:16 编辑
嚴重恭喜！

回复
派
2009年05月1日 15:01 编辑
恭喜！恭喜！

回复
Benjamin
2009年05月1日 14:25 编辑
期待早日见到《我和 LabVIEW》。

回复
